README.md

```diff
@@ -5,37 +5,54 @@ This implementation assumes that you have at least Asterisk 12.0.0 running. This
-        {"42":                                                                   
-          {                                                                      
-            "stations": [                                                        
-            {"endpoint": "SIP/phone1"                                            
-            },                                                                   
-            {"endpoint": "SIP/phone2"}                                           
-            ]                                                                    
-          }                                                                      
-        }                                                                        
-      ]                                                                          
-    }                                                                            
-  }                                                                              
+    {                                                                              
+      "42": {                                                             
+        "trunks": [                                                                
+          "42-A"
+        ]                                                                      
+        "stations": [                                                        
+          "SIP/phone1",
+          "SIP/phone2"                                            
+        ]                                                                
+      }                                                                            
+    }                                                                              
   ]                                                                              
 }
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Right now, however, this application only supports one trunk per sharedExtension, and the sharedExtension MUST have the same name as the trunk. The support for multiple trunks (such as 42A and 42B) will come later.
+Right now, however, this application only supports one trunk per sharedExtension, it can have any name, however, but the SIP trunk must appear in sip.conf, like below: 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+[42-A]
+type = peer
+host = 0.0.0.0
+username = 42-A
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You must also have a dialplan extension in extensions.conf that leads to the application (must have same name as what application is being started in the code) and that has an argument to represent the SLA bridge to reach.  An example is below:
+You must also have a dialplan extension in extensions.conf that leads to the application (must have same name as what application is being started in the code) and that has an argument to represent the SLA bridge to reach.  An example with a device state hint is below:
```

Comment:
Get in the habit of only using one space between sentences. I'd also remove the paranthetical aside, as it isn't a paranthetical. It is necessary to have the same name as the application name in the node.js app.